README.md

```diff
@@ -405,7 +405,6 @@ Local project package:
 ```bash
 rm -rf node_modules dist # use rmdir on Windows
 npm install --save-dev @angular/cli@latest
-npm install
```

Comment:
This was added in https://github.com/angular/angular-cli/pull/2870 to address https://github.com/angular/angular-cli/issues/2410#issuecomment-255880974, I don't know if it's still needed.